internal/compiler/wasm/optimizations.go

```diff
@@ -242,3 +247,44 @@ func reach(cg map[uint32][]uint32, keep map[uint32]struct{}, node uint32) {
 		}
 	}
 }
+
+// skipElem determines if a function in the table is really required:
+// We'll exclude anything with a prefix of "re2::" if none of the known
+// entrypoints into re2 are used.
+func (c *Compiler) skipElem(keep map[uint32]struct{}, idx uint32) bool {
+	if c.usesRE2(keep) {
+		c.debug.Printf("using re2, not skipping its elements")
```

Comment:
Is this debug message going to be useful? We're going to see the same message for every element if the policy relies on re2. I'm tempted to remove it.